### ISSUE TITLE
학교 축제시즌에 날짜와 시간을 맞춰서 가능한 사람들끼리 Meetup을 가져보는건 어떨까요?

### ISSUE THREAD

**Issue by ghost** [ssuoss/HelloSSU#6]
안녕하세요 김종찬입니다.

레포도만들고 아이디어도내고 그런 아이디어를 기반으로 작은 발걸음이나마 @unstabler 님과 제가 그 시작을 알리는 프로젝트를 올려놓은 상황입니다.

이 그룹에서 좀 더 많은 사람들이 기술적인 경험을 교류하고 함께 삽질을 해보는 것에 앞서서 실제로 만나서 이것저것 이야기를 나눠 볼 수 있는 시간들이 있으면 좋겠다는 생각이 들더군요.

그리고 이곳에 현재 참여하신분들은 제 주변에 이미 알고있던 분들이나, 현재 학교에 재학하고 계신분들만 계신것이 아닙니다. Line이나 Kakao에서 일하고계시는 졸업생 선배님들도 계시기도하구요.

이런 다양한사람들이 초면이지만 만나서 우리학교 동문 SW 엔지니어들이 해볼만한게 뭐가있을지, 재밌는게 뭐가있을지 이야기 해볼수있는 시간이 있으면 좋겠습니다.ㅎㅎ

자유롭게 의견주세요~

**Comment by urangurang** [comment]
좋아요~
(지나가던 언론홍보학과 1인입니다.)
만나서 어떤거 개발하면 좋을지 이야기 나눠도 좋을 것 같습니다 ㅎ

**Comment by SSUHan** [comment]
이곳이 생기고 난 시점이 곧 학교 축제와 맞물린다는점이 참 멋진것 같습니다.
3일동안 진행되는 학교 축제 중 적절한 날짜에 미팅을 가진다면 정말 좋을 것 같습니다.
첫날은 어수선할테고.. 둘째날?은 어떤가요??

**Comment by ghost** [comment]
네. 개발자들에게 오프라인에서 만나자고 하면 흔히들 좋은 반응이 안나오는 경우가 많은데..
단 두분이라도 이렇게 반응주셔서 감사합니다.

사실 원격상으로든 Offline에서 만나서하던 일단은 만나서 이야기를하고 놀아야지 뭘 맘을 먹고 시작 할 수 있으니까요.

저도 갑작스럽게 휴가를 써서 나갈수있다면 축제 2일차인 10월 1일부터 나갈수있는 상황입니다.

단 3명이라도 만나서 이런저런 이야기를 나누고 뭔가를 주도해갈수있으면 이 인원으로도 나쁘지 않겠네요.

## 

Ministry of National Defence R&D Engineer

Jongchan Kim

+82-10-9781-0031 (Mobile)

**Comment by wikay** [comment]
참 좋은것 같습니다! 관심있는 분야든 어떤거것이든 같이 모임을 갖을 수 잇다는게 참 좋은 같군요!
2015. 9. 23. 오후 10:53에 "Ace Kim" notifications@github.com님이 작성:

**Comment by Luavis** [comment]
+1

**Comment by ghost** [comment]
네. 그러면 10월 1일 저녁 6시쯤부터 저녁먹고 축제도 즐기고 술마시면서 이런저런 이야기를 나눠보고 싶은분들은
각자 일정 잘 확인하시고 https://www.facebook.com/FlashGuGu로 편하게 누구나 메시지 날려주세요.

**Comment by unstabler** [comment]
+1 저도 가고 싶지만 회사 일이.. ㅜㅜ
